### PR TITLE
Add OSVDB-132234 for rack-attack

### PR DESCRIPTION
**Pull request by reedloden**
https://github.com/kickstarter/rack-attack/releases/tag/v4.3.1
https://github.com/kickstarter/rack-attack/commit/76c2e3143099d938883ae5654527b47e9e6a8977

### CODE REVIEW

File: gems/rack-attack/OSVDB-132234.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+gem: rack-attack
+framework: rails
```
Comment (reedloden) [inline gems/rack-attack/OSVDB-132234.yml]: Is my understanding of the `framework` field correct here? Using it to mean only when used with Rails are you affected by this issue...

Comment (JuanitoFatas) [inline gems/rack-attack/OSVDB-132234.yml]: Shouldn't have :blush: ? From README.md:

> framework [String] (optional): Name of framework gem belongs to.
> [Schema](https://github.com/rubysec/ruby-advisory-db#schema)

Comment (reedloden) [inline gems/rack-attack/OSVDB-132234.yml]: Good catch. We should fix `twitter-bootstrap-rails` then.

**Comment by reedloden** [comment]
@JuanitoFatas / @VanessaHenderson, any other comments, or looks good? :)

**Comment by VanessaHenderson** [comment]
Other than the platform comment it LGTM to me! ;)

**Comment by skorth** [comment]
Looks good :+1:

**Comment by reedloden** [comment]
Thanks!

**Pull request merged**